mobile/src/libs/packages/notification/libs/types/notification-payload.ts

```diff
@@ -0,0 +1,9 @@
+import { type NotificationType } from '#libs/enums/enums';
```

Comment:
Create types.ts file in this folder and re-export types from there. Do the same for enums